File: Prision/src/main/java/com/gkzxhn/prison/view/ISettingView.java
package com.gkzxhn.prison.view;

import com.gkzxhn.prison.entity.VersionEntity;

/**
 * Created by Raleigh.Luo on 18/3/28.
 */

public interface ISettingView extends IBaseView{
    void updateVersion(VersionEntity version);
    void updateFreeTime(int time);
}

File: Prision/src/main/java/com/gkzxhn/prison/view/ISettingView.kt
package com.gkzxhn.prison.view

import com.gkzxhn.prison.entity.VersionEntity

/**
 * Created by Raleigh.Luo on 18/3/28.
 */

interface ISettingView : IBaseView {
    fun updateVersion(version: VersionEntity?)
    //网络状态
    fun networkStatus(isConnected: Boolean)
}
